packages/rehype-mathjax/types/index.d.ts

```diff
@@ -0,0 +1,25 @@
+import {Plugin} from 'unified'
+
+// Should be ported back to MathJax repo
+// http://docs.mathjax.org/en/latest/options/output/svg.html#the-configuration-block
+interface MathJaxSVGOptions {
+  scale: number
+  minScale: number
+  mtextInheritFont: boolean
+  merrorInheritFont: boolean
+  mathmlSpacing: boolean
+  skipAttributes: {[index: string]: boolean}
+  exFactor: number
+  displayAlign: 'left' | 'center' | 'right'
+  displayIndent: string
+  fontCache: 'local' | 'global'
+  localID: string | null
+  internalSpeechTitles: true
+  titleID: number
+}
```

Comment:
MathJax has typings https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/mathjax/index.d.ts which may be able to be referenced.

Also this package has three entrypoints https://github.com/remarkjs/remark-math/tree/82aeb9c3231070c15fc8ea6fa429ffd8677e54b3/packages/rehype-mathjax#rehypeuserehypemathjax-options

Reply:
Looking around `@types/mathjax` but couldn't find SVGOptions there. So we have to define it after all.

Reply:
@ChristianMurphy Added types for chtml and browser in https://github.com/remarkjs/remark-math/pull/48/commits/7ee8c0434b35b15f10db5b8f5c8f6ad1115c3202